packages/core/core/src/requests/EntryRequest.js

```diff
@@ -248,7 +248,9 @@ export class EntryResolver {
         if (!allTargetsHaveSource && pkg.source != null) {
           let pkgSources = Array.isArray(pkg.source)
             ? pkg.source
-            : [pkg.source];
+            : typeof pkg.source === 'string'
+            ? [pkg.source]
+            : [];
```

Comment:
Are there tests somewhere for this change?

Reply:
Well, it's not really a change, this was done to appease Flow...